packages/runtime/src/utils.ts

```diff
@@ -29,3 +29,13 @@ export function autoPrefixer(style: CSSStyleDeclaration): Postprocessor {
       e[0] = autoPrefix(e[0])
   })
 }
+
+let textareaDecoder: HTMLTextAreaElement
+export function decodeHtml(html: string): string {
+  if (!textareaDecoder)
+    textareaDecoder = document.createElement('textarea')
+  textareaDecoder.innerHTML = html
+  html = textareaDecoder.value
+  textareaDecoder.innerHTML = ''
+  return html
+}
```

Comment:
Performance wise, unsure. Feels hacky enough due to SO search attributing this would lead to XSS injection. However most of the context was about dom manipulation; not string processing.

Reply:
I guess we better to use a real decode function for htis.

Reply:
I tried the build using he package, the runtime builds go up by ~88kb.
```
    Directory: C:\x\dev\code\unocss\packages\runtime\builds\he


Mode                 LastWriteTime         Length Name
----                 -------------         ------ ----
-a----         7/30/2022  10:26 AM         228663 attributify.global.js
-a----         7/30/2022  10:26 AM         104165 core.global.js
-a----         7/30/2022  10:26 AM         234278 full.global.js
-a----         7/30/2022  10:26 AM         165872 mini.global.js
-a----         7/30/2022  10:26 AM         225962 uno.global.js


    Directory: C:\x\dev\code\unocss\packages\runtime\builds\textarea


Mode                 LastWriteTime         Length Name
----                 -------------         ------ ----
-a----         7/30/2022  10:24 AM         141061 attributify.global.js
-a----         7/30/2022  10:24 AM          16569 core.global.js
-a----         7/30/2022  10:24 AM         146676 full.global.js
-a----         7/30/2022  10:24 AM          78292 mini.global.js
-a----         7/30/2022  10:24 AM         138360 uno.global.js
```

While the core decode function is small, the size increase mostly come from the [named entities](https://github.com/mathiasbynens/he/tree/master/data). If this is too much, I'd vouch for simple regex replace only for important entities: `& < >` then add other named entities as the issue grow. Within unocss runtime, we are not manipulating dom (well, aside from the main `style` element) so I think it is unlikely to introduce XSS even with incomplete decode function.

Reply:
Alternative implementations are in linked PRs below